packages/quantic/cypress/integration/standalone-search-box/standalone-search-box-selectors.ts

```diff
@@ -0,0 +1,26 @@
+import {ComponentSelector, CypressSelector} from '../common-selectors';
+
+export const standaloneSearchBoxComponent = 'c-quantic-standalone-search-box';
+
+export interface StandaloneSearchBoxSelector extends ComponentSelector {
+  input: () => CypressSelector;
+  suggestionList: () => CypressSelector;
+  clearButton: () => CypressSelector;
+  searchIcon: () => CypressSelector;
+  searchButton: () => CypressSelector;
+}
+
+export const StandaloneSearchBoxSelectors: StandaloneSearchBoxSelector = {
+  get: () => cy.get(standaloneSearchBoxComponent),
+  input: () => StandaloneSearchBoxSelectors.get().find('input[type="search"]'),
+  suggestionList: () =>
+    StandaloneSearchBoxSelectors.get().find(
+      'c-quantic-search-box-suggestions-list li'
+    ),
+  clearButton: () =>
+    StandaloneSearchBoxSelectors.get().find('.slds-input__icon_right'),
+  searchIcon: () =>
+    StandaloneSearchBoxSelectors.get().find('.slds-input__icon_left'),
```

Comment:
These selectors are kind of generic. Is there an attribute we could add to make them more specific and robust?